master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -2,41 +2,64 @@
 title: Kubeadm Hosted Install
 ---
 
-This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+This document outlines how to install Calico on a kubeadm cluster.
 If you have already built your cluster with kubeadm, please review the
 [Requirements / Limitations](#requirements--limitations) at the bottom of
 this page. It is likely you will need to recreate your cluster with the
 `--pod-network-cidr` and `--service-cidr` arguments to kubeadm.
 
-Users who have deployed their own etcd cluster outside of kubeadm should
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
+You can easily create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
-You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+## Installation
 
+The below manifests are compatible with kubeadm clusters initialized with a
```

Comment:
I'd suggest moving the line above ("You can easily create . . .") down to be the first sentence in this section, since it talks about installing a kubeadm cluster.

Would also suggest rewording of this sentence so that it doesn't sound so much like there's a limitation in Calico that the 192.168 cidr be used.  Maybe something like this?

"Note that `--pod-network-cidr` is not always required when creating a kubeadm cluster.  When it is specified, the value must match the IPV4_POOL_CIDR in the Calico manifest.  Calico uses `192.168.0.0/16` by default.

For example:

```
kubeadm init --pod-network-cidr=192.168.0.0/16
```
"

^ That wording isn't great but hopefully shows the direction I'm thinking we should go.

Reply:
We require the `--pod-network-cidr` to be set for both etcd and kdd correct? If not then I should probably also move that statement to the Kubernetes section.

Reply:
> Note that --pod-network-cidr is not always required when creating a kubeadm cluster

Other than that, yeah the same statement applies to both manifests.